e2e/sidecars/nginx/Dockerfile

```diff
@@ -1,2 +1,8 @@
-FROM nginx
-COPY nginx.conf /etc/nginx/nginx.conf
+FROM nginx:1.15-alpine
```

Comment:
https://serverfault.com/questions/577370/how-can-i-use-environment-variables-in-nginx-conf